### PR TITLE
renamed replication, added shard and proxy-threads

### PR DESCRIPTION
**Pull request by j8-redis**
### What does this PR do?

adds two dashboards and renames a thrid

### Motivation

clients have asked for this functionality

### Review checklist

meaningful title -> more redis enterprise dashoards

feature requires no new tests

history is clean

### Additional Notes

Those are the only changes

### CODE REVIEW

File: redis_enterprise/assets/dashboards/redis_enterprise_proxy-threads.json [hunk 1]
```diff
@@ -0,0 +1,528 @@
+{
+  "title": "Redis Enterprise - Proxy Threads",
+  "description": "[[suggested_dashboards]]",
```
Comment (eho1307) [inline redis_enterprise/assets/dashboards/redis_enterprise_proxy-threads.json:3]: Could you add a description in here?

File: redis_enterprise/assets/dashboards/redis_enterprise_shard.json [hunk 2]
```diff
@@ -0,0 +1,1579 @@
+{
+  "title": "Redis Enterprise - Shard",
+  "description": "[[suggested_dashboards]]",
```
Comment (eho1307) [inline redis_enterprise/assets/dashboards/redis_enterprise_shard.json:3]: Could you add a description in here?

**Review by eho1307 — CHANGES_REQUESTED** [review]

File: redis_enterprise/assets/dashboards/redis_enterprise_proxy-threads.json (hunk 1 above)
Comment (j8-redis) [reply redis_enterprise/assets/dashboards/redis_enterprise_proxy-threads.json]: done for all

File: redis_enterprise/assets/dashboards/redis_enterprise_shard.json (hunk 2 above)
Comment (j8-redis) [reply redis_enterprise/assets/dashboards/redis_enterprise_shard.json]: done for all

File: redis_enterprise/assets/dashboards/redis_enterprise_proxy.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "title": "Redis Enterprise - Proxy",
-  "description": "[[suggested_dashboards]]",
+  "description": "Redis Enterprise Proxy presents sa single, usable endpoint that hides topological complexity",
```
Comment (eho1307) [inline redis_enterprise/assets/dashboards/redis_enterprise_proxy.json:3]: Is 'sa' intentional here?

File: redis_enterprise/assets/dashboards/redis_enterprise_shard.json [hunk 4]
```diff
@@ -0,0 +1,1579 @@
+{
+  "title": "Redis Enterprise - Shard",
+  "description": "[Redis Enterprise Shards are the individual database partitions",
```
Comment (eho1307) [inline redis_enterprise/assets/dashboards/redis_enterprise_shard.json:3]:
```suggestion
  "description": "Redis Enterprise Shards are the individual database partitions",
```

**Review by eho1307 — CHANGES_REQUESTED** [review]

File: redis_enterprise/assets/dashboards/redis_enterprise_proxy.json (hunk 3 above)
Comment (j8-redis) [reply redis_enterprise/assets/dashboards/redis_enterprise_proxy.json]: "... presents a single, ..."

**Review by eho1307 — APPROVED** [review]

**Review by iliakur — APPROVED** [review]

File: redis_enterprise/assets/dashboards/redis_enterprise_proxy-threads.json [hunk 5]
```diff
@@ -0,0 +1,528 @@
...
+        "background_color": "vivid_blue",
+        "show_title": true,
+        "type": "group",
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 7104524980730882,
+            "definition": {
+              "type": "note",
+              "content": "Redis Software allows multiple databases to be created. Each database gets an endpoint (a unique URL and port on the FQDN). This endpoint receives all the traffic for all operations for that database. By default, Redis Software binds this database endpoint to one of the proxies on a single node in the cluster. This proxy becomes an active proxy and receives all the operations for the given database. (note that if the node with the active proxy fails, a new proxy on another node takes over as part of the failover process automatically).",
```
Comment (buraizu) [inline redis_enterprise/assets/dashboards/redis_enterprise_proxy-threads.json:18]:
```suggestion
              "content": "Redis Software allows multiple databases to be created. Each database gets an endpoint (a unique URL and port on the FQDN). This endpoint receives all the traffic for all operations for that database. By default, Redis Software binds this database endpoint to one of the proxies on a single node in the cluster. This proxy becomes an active proxy and receives all the operations for the given database. (Note that if the node with the active proxy fails, a new proxy on another node takes over as part of the failover process automatically.)",
```

**Review by buraizu — APPROVED** [review]
Approving with a minor update requested

File: redis_enterprise/manifest.json [hunk 6]
```diff
@@ -57,8 +57,10 @@
       "redis-enterprise-overview": "assets/dashboards/redis_enterprise_overview.json",
       "redis-enterprise-database": "assets/dashboards/redis_enterprise_database.json",
       "redis-enterprise-node": "assets/dashboards/redis_enterprise_node.json",
-      "redis-enterprise-replication": "assets/dashboards/redis_enterprise_replication.json",
-      "redis-enterprise-proxy": "assets/dashboards/redis_enterprise_proxy.json"
+      "redis-enterprise-active-active": "assets/dashboards/redis_enterprise_active-active.json",
+      "redis-enterprise-proxy": "assets/dashboards/redis_enterprise_proxy.json",
+      "redis-enterprise-proy-threads": "assets/dashboards/redis_enterprise_proxy-threads.json",
```
Comment (bgoldberg122) [inline redis_enterprise/manifest.json:62]: Should this be `proxy-threads?
```suggestion
      "redis-enterprise-proxy-threads": "assets/dashboards/redis_enterprise_proxy-threads.json",
```

File: redis_enterprise/manifest.json [hunk 7]
```diff
@@ -57,8 +57,10 @@
       "redis-enterprise-overview": "assets/dashboards/redis_enterprise_overview.json",
       "redis-enterprise-database": "assets/dashboards/redis_enterprise_database.json",
       "redis-enterprise-node": "assets/dashboards/redis_enterprise_node.json",
-      "redis-enterprise-replication": "assets/dashboards/redis_enterprise_replication.json",
-      "redis-enterprise-proxy": "assets/dashboards/redis_enterprise_proxy.json"
+      "redis-enterprise-active-active": "assets/dashboards/redis_enterprise_active-active.json",
```
Comment (bgoldberg122) [inline redis_enterprise/manifest.json:60]: Changing the file name and Title is fine, but please keep the same `key` here
```suggestion
      "redis-enterprise-replication": "assets/dashboards/redis_enterprise_active-active.json",
```

**Review by bgoldberg122 — CHANGES_REQUESTED** [review]

File: redis_enterprise/manifest.json (hunk 6 above)
Comment (j8-redis) [reply redis_enterprise/manifest.json]: good catch

**Review by estherk15 — APPROVED** [review]

**Review by moreauowen — APPROVED** [review]

**Pull request merged**